pkg/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/kubemon.go

```diff
@@ -73,7 +72,7 @@ func (mod KubernetesMonitoringModifier) getInitContainers() []corev1.Container {
 			Args:            []string{"-c", k8scrt2jksPath},
 			VolumeMounts:    volumeMounts,
 			Resources:       mod.capability.Properties().Resources,
-			SecurityContext: GetSecurityContext(readOnlyRootFs),
+			SecurityContext: GetSecurityContext(true),
```

Comment:
do we ever use `GetSecurityContext(false)` ?
if not, we don't need to pass in anything

Reply:
Yes we do, will leave it as is